client/src/lib/session/hooks/useAdjustVolume.ts

```diff
@@ -0,0 +1,23 @@
+import {useCallback} from 'react';
+import {Platform} from 'react-native';
+import {VolumeManager} from 'react-native-volume-manager';
+
+const useAdjustVolume = () => {
+  return useCallback(async () => {
+    if (Platform.OS === 'android') {
+      const {music, call} = await VolumeManager.getVolume();
+
+      if (call !== music) {
+        VolumeManager.setVolume(Math.max(call ?? 0, music ?? 0), {
+          type: 'music',
+          showUI: false,
```

Comment:
I was thinking that maybe we should show this, but only when the audio actually starts playing. Not a biggie, maybe for the next iteration...

Reply:
Show that we adjust it you mean? Yea, I was thinking let's see if this helps with the bug and then refine it a bit